includes/Helpers/LogHelper.php

```diff
@@ -91,49 +85,42 @@ public static function getRequestLogsWithComments($requestId, PdoDatabase $db, S
         return $items;
     }
 
-    /**
-     * Summary of getLogDescription
-     *
-     * @param Log $entry
-     *
-     * @return string
-     */
-    public static function getLogDescription(Log $entry)
+    public static function getLogDescription(Log $entry): string
     {
-        $text = "Deferred to ";
+        $text = 'Deferred to ';
```

Comment:
I see that you're doing this wholesale standardization of double quotes to single quotes, but there's a few files in this PR where that doesn't appear to have been done, such as `Logger.php`. Is there a reason for this? I wonder if it would be better to split off the quotes normalization into a separate PR for ease of review/consistency?

Reply:
I've removed the quote changes from the PR for now